app/models/project.rb

```diff
@@ -95,9 +97,27 @@ def fork(user)
     forked_project.update!(
       view: 1, author_id: user.id, forked_project_id: id, name: name
     )
+    @project = Project.find(id)
+    if @project.author != user
+      ForkNotification.with(user: user, project: @project).deliver_later(@project.author)
+    end
     forked_project
   end
 
+  acts_as_notifiable :users,
+  # Notification targets as :targets is a necessary option
+  targets: lambda { |project, _key|
+    [project.forked_project.author]
+  },
+  notifier: :author,
+  printable_name: lambda { |project|
+    "forked your project \"#{project.name}\""
```

Comment:
Can we make this i18n compatiable ?

Reply:
This is the configuration of `ActivityNotification`, after merging, this is going to be removed.
No need!